webapp/channels/src/types/external/react-bootstrap.d.ts

```diff
@@ -11,6 +11,7 @@ import type {OverlayTriggerProps} from 'react-bootstrap';
 export interface AdditionalOverlayTriggerProps extends React.ComponentPropsWithRef<typeof OverlayTriggerProps> {
 
     className?: string;
+    overlay: any;
```

Comment:
Do we not have a type for this?

Reply:
Ditto with this. There is a correct type, but between the outdated version of React Bootstrap and our changes, I'm not wanting to bother finding what that is